lxd/instance/drivers/driver_qemu.go

```diff
@@ -6148,30 +6148,32 @@ func (d *qemu) Info() instance.Info {
 	}
 
 	if !shared.PathExists("/dev/kvm") {
-		data.Error = fmt.Errorf("KVM support is missing")
+		data.Error = fmt.Errorf("KVM support is missing (no /dev/kvm)")
 		return data
 	}
 
 	err := util.LoadModule("vhost_vsock")
 	if err != nil {
-		data.Error = fmt.Errorf("vhost_vsock kernel module not loaded: %w", err)
+		data.Error = fmt.Errorf("vhost_vsock kernel module not loaded")
 		return data
 	}
 
 	hostArch, err := osarch.ArchitectureGetLocalID()
 	if err != nil {
-		data.Error = fmt.Errorf("Failed getting architecture")
+		logger.Errorf("Failed getting CPU architecture: %v", err)
```

Comment:
Why are some errors logged and others not? Also if we are keeping them it would be good to indicate in the error they are regarding VM support.

Reply:
I'm logging the unexpected ones, so those which should never really happen and aren't just about some missing dependencies.